test/component/scheduler/perf/util.go

```diff
@@ -109,54 +110,60 @@ func makeNodes(c client.Interface, nodeCount int) {
 	}
 }
 
-// makePods will setup specified number of scheduled pods.
-// Currently it goes through scheduling path and it's very slow to setup large number of pods.
-// TODO: Setup pods evenly on all nodes and quickly/non-linearly.
-func makePods(c client.Interface, podCount int) {
-	glog.Infof("making %d pods", podCount)
-	basePod := &api.Pod{
+func makePodSpec() api.PodSpec {
+	return api.PodSpec{
```

Comment:
Actually, Labels are part of ObjectMeta, which I'm setting in makePodsFromRC. But sure - good catch.
Added there.